kinto/plugins/history/__init__.py

```diff
@@ -12,7 +12,14 @@ def includeme(config):
     # Activate end-points.
     config.scan('kinto.plugins.history.views')
 
+    # If StatsD is enabled, monitor execution time of listener.
+    if config.registry.statsd:
+        key = 'listeners.history'
```

Comment:
Maybe it should be `plugins.history` here instead of `listeners.X`, since we use those for listeners configured via `.ini` files?